### PR TITLE
Changed the filter dialogues for tasks and overrides

### PR DESCRIPTION
**Pull request by saberlynx**
- Changed the filter dialogues for tasks and overrides
- added tasktrendgroup component.

**Checklist**:

- [x] Tests
- [x] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/web/components/powerfilter/__tests__/tasktrendgroup.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+import TaskTrendGroup from 'web/components/powerfilter/tasktrendgroup';
+import Filter from 'gmp/models/filter';
+
+const openSelectElement = element => {
+  const openButton = getByTestId(element, 'select-open-button');
+  fireEvent.click(openButton);
+};
+
+const getItemElements = baseElement => {
```
Comment (bjoernricks) [inline gsa/src/web/components/powerfilter/__tests__/tasktrendgroup.js]: Could you import openSelectElement and getItemElements from web/components/form/__tests__/select instead?

File: gsa/src/web/components/powerfilter/tasktrendgroup.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ */
+
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+import _ from 'gmp/locale';
+
+import {isDefined} from 'gmp/utils/identity';
+
+import PropTypes from '../../utils/proptypes.js';
```
Comment (bjoernricks) [inline gsa/src/web/components/powerfilter/tasktrendgroup.js]:
```suggestion
import PropTypes from 'web/utils/proptypes';
```

File: gsa/src/web/components/powerfilter/tasktrendgroup.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+import FormGroup from '../form/formgroup.js';
```
Comment (bjoernricks) [inline gsa/src/web/components/powerfilter/tasktrendgroup.js]:
```suggestion
import FormGroup from 'web/components/form/formgroup';
```

File: gsa/src/web/components/powerfilter/tasktrendgroup.js (hunk 2 above)
Comment (saberlynx) [reply gsa/src/web/components/powerfilter/tasktrendgroup.js]: haha should have remembered to change these

**Pull request merged**